tests/Unit/Options/Python/Test_ExtractInputSourceYAMLFromH5.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+from spectre.Options.ExtractInputSourceYAMLFromH5 import *
+
+import spectre.Informer as spectre_informer
+import unittest
+import os
+import h5py
+
+# For Py2 compatibility
+try:
+    unittest.TestCase.assertRaisesRegex
+except AttributeError:
+    unittest.TestCase.assertRaisesRegex = unittest.TestCase.assertRaisesRegexp
+
+
+class TestExtractInputSourceYAMLFromH5(unittest.TestCase):
```

Comment:
sorry, no I meant compare python reading the stand-alone yaml file with python reading what you get by embedding that into an h5 file and then extracting it

Reply:
If I do that, I'll need to commit the yaml file that generated the test h5 data...if that's ok, would you suggest committing it next to the test h5 file or some place else?